operator/v1/types.go

```diff
@@ -126,6 +130,7 @@ type OperatorStatus struct {
 	ReadyReplicas int32 `json:"readyReplicas"`
 
 	// generations are used to determine when an item needs to be reconciled or has changed in a way that needs a reaction.
+	// +listType=atomic
```

Comment:
map with keys group, resource, namespace, name

Reply:
Rerolled to do this with 0337baf10 -> 800141644.  That drops the follow-up type stiffening for now, so I have a clean validation CI failure to base the `OperatorCondition` and now `GenerationStatus` changes on.